src/icons/README.md

```diff
@@ -1,12 +1,14 @@
 # qBittorrent Icons
-
-Initial `qbt-theme` icons created by Bert Verhelst (<verhelstbert@gmail.com>).
-
-Icons are based on the `Font-Awesome` icon-set: [link](http://fontawesome.io/icons/).
-
-If you need to add an icon that qBittorrent does not already use, you can take an icon from the SVG fork of `Font-Awesome`: [link](https://github.com/encharm/Font-Awesome-SVG-PNG).
-
-
-## Optimizing SVG
-
-Use [svgcleaner](https://github.com/RazrFalcon/svgcleaner)
+------------------------------------------
+
+## Previous
+Old icons were created by Bert Verhelst (<verhelstbert@gmail.com>).
+Icons were based on the [Font-Awesome](http://fontawesome.io/icons/) icon set.
```

Comment:
It would be great if you can help update it to https:
```suggestion
Icons were based on the [Font-Awesome](https://fontawesome.com/icons) icon set.
```